### DISCUSSION TITLE
ColorInput styling

### DISCUSSION THREAD

**Discussion by ghost** [mantinedev/mantine#2199] [Q&A] [answered]
How do I style the ColorInput element to get the view shown on the right in the image?  No text part in the input field and the color (or color + opacity pattern) shown in the circle on the left fills the entire square.  

![image](https://user-images.githubusercontent.com/2243686/185851473-5c363ac7-6503-479d-bad5-5064756271af.png)

**Comment by rtivital** [comment, marked as answer]
You should not use ColorInput for that. Use Popover, ColorPicker and ColorSwatch components to build your own color picker.

**Comment by ghost** [comment]
Thanks.  Will look into that.
